lib/worker/src/rpc.rs

```diff
@@ -148,7 +158,17 @@ impl<E: EVMClient + 'static> publications::Server for Publications<E> {
                 return Err(Error::failed("namespace not found".into()));
             }
 
-            let upload_type = UploadType::Simple(Media::new(filename.clone()));
+            let upload_type = UploadType::Multipart(Box::new(Object {
```

Comment:
we received the timestamp and pass as metadata to GCS

Reply:
the cli will send current timestamp if the user doesn't provide the timestamp, right?

Reply:
yeah 👍